app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Any
+
+
+def cache(func) -> Any:
+    data_save = {}
+
+    def inner(*args: tuple[(int, float, tuple, str)]) -> Any:
```

Comment:
If you want have a consistency with typings, do not pass `Any` for determined result